mkdocs.yml

```diff
@@ -0,0 +1,71 @@
+site_name: Probabilistic Programming in Haskell
+theme:
+  name: material
+  features:
+    # - announce.dismiss
+    - content.code.annotate
+    # - content.tabs.link
+    - content.tooltips
+    # - header.autohide
+    # - navigation.expand
+    # - navigation.indexes
+    # - navigation.instant
+    # - navigation.prune
+    - navigation.sections
+    # - navigation.tabs
+    # - navigation.tabs.sticky
+    - navigation.top
+    - navigation.tracking
+    - search.highlight
+    - search.share
+    - search.suggest
+    - toc.follow
+  palette:
```

Comment:
Nice! I wonder if this will work with bright and dark mode of the users' browsers.